backend/tests/unit/apps/owasp/models/crp/certificate_test.py

```
@@ -0,0 +1,114 @@
+from unittest.mock import MagicMock, patch
```

Ruff (0.16.0)

[warning] File `backend/tests/unit/apps/owasp/models/crp/certificate_test.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)